### PR TITLE
Add deployment docs

### PR DESCRIPTION
**Pull request by otterbotter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,15 @@ ES6 and Webflow.
 
     yarn run import-webflow zipfile.zip
 
+
+## Deployment 
+
+All updates to `master` will automatically deploy to Netlify.
```
Comment (schalkneethling) [inline README.md:14]: This is production correct? If so, then I think we should be even more explicit.

```suggestion
All updates to `master` will automatically deploy to the production instance on Netlify.
```

File: README.md [hunk 2]
```diff
@@ -8,6 +8,15 @@ ES6 and Webflow.
...
+
+> NOTE: Do not merge into master if you are not ready for it to be deployed!
```
Comment (schalkneethling) [inline README.md:16]:
```suggestion
> NOTE: Do not merge into master if you are not ready for it to be deployed to production! 
```

File: README.md [hunk 3]
```diff
@@ -8,6 +8,15 @@ ES6 and Webflow.
...
     yarn run import-webflow zipfile.zip
 
+
+## Deployment 
+
+All updates to `master` will automatically deploy to Netlify.
+
+> NOTE: Do not merge into master if you are not ready for it to be deployed! 
+If you want to merge a breaking change on the backend and co-ordinate with the frontend deployment, 
+merge the backend and deploy before merging the frontend as deploying happens automatically.
```
Comment (schalkneethling) [inline README.md:18]:
```suggestion
merge and deploy the backend changes _before_ merging the frontend changes.
```

**Review by schalkneethling — CHANGES_REQUESTED** [review]
A couple of suggestions.

**Comment by otterbotter** [comment]
Thanks for the suggestions @schalkneethling

**Review by schalkneethling — APPROVED** [review]

**Pull request merged**